spec.bs

```diff
@@ -3247,6 +3250,74 @@ a [=list=] of [=interest groups=] |bidIgs|, and a [=reporting context map=]
 
 </div>
 
+<div algorithm>
+To <dfn>handle server response private aggregation fields</dfn> given a [=server auction response=]
+|response|, a [=reporting context=] |reportingContext|, and a [=reporting bid key=] |reportingId|:
+
+1. [=Assert=] that these steps are running [=in parallel=].
+1. [=Commit server response private aggregation contributions=] given |response|'s
+  [=server auction response/component win private aggregation contributions=], |reportingContext|,
+  and |reportingId|.
+1. [=Commit server response private aggregation contributions=] given |response|'s
+  [=server auction response/server filtered private aggregation reserved contributions=],
+  |reportingContext|, and |reportingId|.
+1. [=Commit server response private aggregation contributions=] given |response|'s
+  [=server auction response/server filtered private aggregation non reserved contributions=],
+  |reportingContext|, and |reportingId|.
+
+</div>
+
+<div algorithm>
+To <dfn>commit server response private aggregation contributions</dfn> given a [=map=] from a
+[=tuple=] of ([=reporting bid key=], [=string=]) to a [=list=] of [=on event contribution entries=]
+|contributionsMap|, a [=reporting context=] |reportingContext|, and a [=reporting bid key=]
+|reportingId|:
+
+1. [=map/For each=] (|reportingOrigin|, |coordinator|, |event|) → |contributions| of |contributionsMap|:

```

Comment:
The key type doesn't seem to match up here? (And I don't see how to get either the coordinator or reportingOrigin from reporting bid key)

Reply:
ah yes, forgot to update them. Created a new structure for it since it's used in multiple places, to possibly improve its readability. I'll update the IETF spec to refer to this structure when this is merged.